packages/yew-macro/src/props/component.rs

```diff
@@ -138,11 +159,13 @@ impl Parse for ComponentProps {
         let base_expr = if input.is_empty() {
             None
         } else {
-            let _dots: Dot2 = input.parse()?;
-            Some(input.parse()?)
+            let dot2 = input.parse()?;
+            let expr = input.parse()?;
+            Some(BaseExpr { dot2, expr })
```

Comment:
```suggestion
            Some(input.parse::<BaseExpr>()?)
```
Can parse the type and by doing this with the change above get a nice error message that only makes sense for `BaseExpr` :D